simulator/autoware_auto_msgs_adapter/README.md

```diff
@@ -0,0 +1,96 @@
+# autoware_auto_msgs_adapter
+
+This package is used to convert `autoware_msgs` to `autoware_auto_msgs`.
+
+## Purpose
+
+As we transition from `autoware_auto_msgs` to `autoware_msgs`, we wanted to provide flexibility and compatibility for
+users who are still using `autoware_auto_msgs`.
+
+This adapter package allows users to easily convert messages between the two formats.
+
+## Capabilities
+
+The `autoware_auto_msgs_adapter` package provides the following capabilities:
+
+- Conversion of supported `autoware_msgs` messages to `autoware_auto_msgs` messages.
+- Can be extended to support conversion for any message type pairs.
+- Each instance is designed to convert from a single source message type to a single target message type.
+- Multiple instances can be launched to convert multiple message types.
+- Can be launched as a standalone node or as a component.
+
+## Usage
+
+Customize the adapter configuration by replicating and editing the `adapter_control.param.yaml` file located
+in the `autoware_auto_msgs_adapter/config` directory. Example configuration:
+
+```yaml
+/**:
+  ros__parameters:
+    msg_type_target: "autoware_auto_control_msgs::msg::AckermannControlCommand"
```

Comment:
I feel like a slash delimited type name is better for users. The ros2 commands and maybe some utility functions uses slashes.

Reply:
You are right, fixed it in: https://github.com/autowarefoundation/autoware.universe/pull/3876/commits/66edca24d3028a10fe4eae95008f80538f14d0f0